### PR TITLE
Add uploadtoftp module.md file

### PR DESCRIPTION
**Pull request by Nipunich**
## Purpose
Add uploadtoftp module.md file and change template details.

### CODE REVIEW

File: docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/uploadtoftp/Module.md [hunk 1]
```diff
@@ -1 +1,92 @@
-# Convert JSON to XML and upload to FTP
+Template for JSON to XML and upload to FTP
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/uploadtoftp/Module.md]:
```suggestion
Template to convert JSON to XML and upload to FTP
```

File: docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/uploadtoftp/Module.md [hunk 2]
```diff
@@ -1 +1,92 @@
...
+$ ballerina new converting-json-to-xml-and-upload-to-ftp
+```
+
+Now navigate into the above module directory you created and run the following command to apply the predefined template you pulled earlier.
+
+```bash
+$ ballerina add -t wso2/uploadtoftp uploadtoftp
+```
+
+This automatically creates restful_service for you inside the `src` directory of your project.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/uploadtoftp/Module.md]: Should be something like converting_json_to_xml_and_upload_to_ftp.

File: docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/uploadtoftp/Module.md [hunk 3]
```diff
@@ -1 +1,92 @@
...
+$ ballerina build uploadtoftp 
+```
+
+This creates the executables. Now run the `uploadtoftp.jar` file created in the above step.
+
+```bash
+$ java -jar target/bin/uploadtoftp.jar
+```
+
+Create an employees.json file with the below payload.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/uploadtoftp/Module.md]: Before this we have to say that the service has started. After that we can create this file and curl it.

File: docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/uploadtoftp/Module.md (hunk 2 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/uploadtoftp/Module.md]: Changed to upload_to_ftp

File: docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/uploadtoftp/Module.md (hunk 3 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/uploadtoftp/Module.md]: It's required to create the file beforehand to curl it.

File: docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/upload_to_ftp/Module.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+$ ballerina new converting-json-to-xml-and-upload-to-ftp
+```
+
+Now navigate into the above module directory you created and run the following command to apply the predefined template you pulled earlier.
+
+```bash
+$ ballerina add -t wso2/upload_to_ftp upload_to_ftp
+```
+
+This automatically creates restful_service for you inside the `src` directory of your project.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/upload_to_ftp/Module.md]: Update the template name here.

Comment (Nipunich) [reply docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/upload_to_ftp/Module.md]: Fixed

**Pull request merged**